Common/src/main/resources/assets/biomeswevegone/lang/ru_ru.json

```
@@ -94,93 +94,93 @@
   "block.biomeswevegone.amaranth": "Амарант",
   "block.biomeswevegone.angelica": "Дудник",
   "block.biomeswevegone.apple_fruit": "Зреющее яблоко",
-  "block.biomeswevegone.araucaria_leaves": "Араукариевая листва",
+  "block.biomeswevegone.araucaria_leaves": "Араукариевые листья",
   "block.biomeswevegone.araucaria_sapling": "Саженец араукарии",
-  "block.biomeswevegone.aspen_bookshelf": "Осиновые книжные полки",
+  "block.biomeswevegone.aspen_bookshelf": "Осиновая книжная полка",
   "block.biomeswevegone.aspen_button": "Осиновая кнопка",
   "block.biomeswevegone.aspen_crafting_table": "Осиновый верстак",
   "block.biomeswevegone.aspen_door": "Осиновая дверь",
   "block.biomeswevegone.aspen_fence_gate": "Осиновая калитка",
   "block.biomeswevegone.aspen_fence": "Осиновый забор",
   "block.biomeswevegone.aspen_hanging_sign": "Осиновая подвесная табличка",
-  "block.biomeswevegone.aspen_leaves": "Осиновая листва",
-  "block.biomeswevegone.aspen_log": "Осиновая древесина",
+  "block.biomeswevegone.aspen_leaves": "Осиновые листья",
+  "block.biomeswevegone.aspen_log": "Осиновое бревно",
   "block.biomeswevegone.aspen_planks": "Осиновые доски",
   "block.biomeswevegone.aspen_pressure_plate": "Осиновая нажимная плита",
   "block.biomeswevegone.aspen_sapling": "Осиновый саженец",
```

Да, забыл. Перевод всех саженцев выглядит как "Саженец ..."
```suggestion
  "block.biomeswevegone.aspen_sapling": "Саженец осины",
```